.github/scripts/release-plugins.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+git config --global user.email "githubbot@gluonhq.com"
+git config --global user.name "Gluon Bot"
+
+CLIENT_MAVEN_REPO_SLUG=gluonhq/client-maven-plugin
+CLIENT_GRADLE_REPO_SLUG=gluonhq/client-gradle-plugin
+
+cd $HOME
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5592359)